### DISCUSSION TITLE
Hi .I'm sorry ,I have a problem.about learndash pay for lesson .so ,when I download your plugin and i active in my site WordPress. It has oneerror from rest api and one error from session .please hlep me

### DISCUSSION THREAD

**Discussion by AmirHosseinGHaniZadeh** [community/community#124658] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi .I'm sorry ,I have a problem.about learndash pay for lesson .so ,when I download your plugin and i active in my site WordPress. It has oneerror from rest api and one error from session .please hlep me
![17161246074063089544270263051759](https://github.com/community/community/assets/170246134/016bb98c-0ec1-4531-8475-44512afc1909)

**Comment by ebndev** [comment]
Hi @AmirHosseinGHaniZadeh,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Discussions category is for conversations related to GitHub Discussions. I have moved this post to General, however, it looks like you're looking for support from a specific project. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
